examples/arkode/C_serial/CMakeLists.txt

```diff
@@ -59,29 +59,30 @@ set(ARKODE_examples
     "ark_kepler\;--stepper SPRK --step-mode fixed --method ARKODE_SPRK_RUTH_3_3 --tf 50 --check-order --nout 1\;exclude-single"
     "ark_kepler\;--stepper SPRK --step-mode fixed --method ARKODE_SPRK_YOSHIDA_6_8 --tf 50 --check-order --nout 1\;exclude-single"
     "ark_kepler\;\;develop"
-    "ark_kpr_mri\;\;develop"
-    "ark_kpr_mri\;0 0.002\;develop"
-    "ark_kpr_mri\;1 0.002\;develop"
-    "ark_kpr_mri\;2 0.005\;develop"
-    "ark_kpr_mri\;3 0.01\;develop"
-    "ark_kpr_mri\;4 0.002\;develop"
-    "ark_kpr_mri\;5 0.002\;develop"
-    "ark_kpr_mri\;6 0.005\;develop"
-    "ark_kpr_mri\;7 0.001 -100 100 0.5 1\;exclude-single"
-    "ark_kpr_mri\;7 0.001\;exclude-single"
-    "ark_kpr_mri\;8 0.001 -100 100 0.5 1\;exclude-single"
-    "ark_kpr_mri\;8 0.001\;exclude-single"
-    "ark_kpr_mri\;9 0.001 -100 100 0.5 1\;exclude-single"
-    "ark_kpr_mri\;9 0.001\;exclude-single"
+    "ark_kpr_mri\;0 1 0.005\;exclude-single"
+    "ark_kpr_mri\;1 0 0.01\;exclude-single"
+    "ark_kpr_mri\;1 1 0.002\;exclude-single"
+    "ark_kpr_mri\;2 4 0.002\;exclude-single"
+    "ark_kpr_mri\;3 2 0.001\;exclude-single"
+    "ark_kpr_mri\;4 3 0.001\;exclude-single"
+    "ark_kpr_mri\;5 4 0.001\;exclude-single"
+    "ark_kpr_mri\;6 5 0.001\;exclude-single"
+    "ark_kpr_mri\;7 2 0.002\;exclude-single"
+    "ark_kpr_mri\;8 3 0.001 -100 100 0.5 1\;exclude-single"
+    "ark_kpr_mri\;9 3 0.001 -100 100 0.5 1\;exclude-single"
+    "ark_kpr_mri\;10 4 0.001 -100 100 0.5 1\;exclude-single"
+    "ark_kpr_mri\;11 2 0.001\;exclude-single"
+    "ark_kpr_mri\;12 3 0.005\;exclude-single"
+    "ark_kpr_mri\;13 4 0.01\;exclude-single"
     "ark_KrylovDemo_prec\;\;exclude-single"
     "ark_KrylovDemo_prec\;1\;exclude-single"
     "ark_KrylovDemo_prec\;2\;exclude-single"
-    "ark_onewaycouple_mri\;\;develop"
+    "ark_onewaycouple_mri\;\;exclude-single"
     "ark_reaction_diffusion_mri\;\;develop"
     "ark_robertson_constraints\;\;exclude-single"
     "ark_robertson_root\;\;exclude-single"
     "ark_robertson\;\;exclude-single"
-    "ark_twowaycouple_mri\;\;develop")
+    "ark_twowaycouple_mri\;\;exclude-single")
```

Comment:
Why did a few of these example have to change to `exclude-single`?

Reply:
Good question.  I seem to recall intermittent single-precision CI failures for these two, but in looking at the code I can't predict why.  I can add them back in and see what happens.